docs/tutorials/12_deuteron_binding_energy.ipynb

```diff
@@ -0,0 +1,1286 @@
+      "text/plain": [
+       "<Figure size 864x576 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.figure(figsize=(12, 8))\n",
```

Comment:
I think it would be nicer, if we can show the expectation values for the various Hamiltonians as subplots, rather than four consecutive plots. This would also reduce the duplicate code in all of these plotting cells.

The same applies for the earlier plot sequences